src/components/Auth/assets/animations.scss

```diff
@@ -0,0 +1,133 @@
+/* stylelint-disable function-parentheses-space-inside */
+:global {
+  .loaderSvg {
```

Comment:
Nice. `onfidoLoaderSvg` would make it even more specific, so we don't end up with conflicts with the host app